### ISSUE TITLE
Print items in centre of frame

### ISSUE THREAD

**Issue by bwpearre** [bwpearre/printimage#4]
Given the prevalence of laser alignment problems, we seem to be seeing cases in which edges get cut off because they're too close to the edge of the FOV. Maybe print things in the centre?

**Comment by Christos-Michas** [comment]
Assume we have a big STL with many metavoxels. Only the metavoxels at the edges will not occupy the whole FOV.
- We only care about centering the metavoxels/pieces that do not take up the whole FOV. For those only, we would have to:
    - move the voxelization within the FOV to center it in the FOV
    - subtract the equivalent # of um from the movement of the stage when moving from the second-to-last to the last metavoxel
- For the other metavoxels, centering won’t have any effect. If there is vignetting, the only solution is to increase the non-vignetting zoom.

After some experimentation with the current system, there is no way to get uniform power throughout the whole FOV when the zoom factor is just 1, so there will always be vignetting when using that factor. If we increase the non-vignetting zoom, then we are zooming in in the center of the FOV, where the power higher and more uniform. To get sufficiently uniform power, we have to go up to 2x zoom. Thus, in the end, if centering the metavoxels is such a pain, we might as well just skip it because we will never use a zoom factor lower than 2x.

**Comment by bwpearre** [comment]
That is true. I'd been wondering about a parallel approach: modify power to reverse vignetting. But it's just yet more calibration. It would be easy to do approximately, but the effect at zoom=1 is severe, so the main problem I see is interacting with the user: the "no you can't do that, how about this" conversation.

Meanwhile, as you say, the only case in which printing in the centre of the frame is useful is for single-XY-voxel items with X > ~400 um and Y < ~200 um or vice versa: it'll only help on objects with a high-ish aspect ratio.

Just moving items to the centre of the frame is easy. Seemed like it wouldn't be very broadly useful, but if it is, please re-open this bug.
